core/sqf/sql/scripts/sqstart

```diff
@@ -220,6 +220,31 @@ function SQCheckOrphanProcesses {
 
 }
 
+function checkKerberos {
+
+   # Check to see if kerberos is enabled in Hadoop
+   cat /etc/hadoop/conf/core-site.xml | while read a; do
```

Comment:
Can you use  HADOOP_CNF_DIR environment variable instead of specifying the fully qualified directory